arch/riscv/Kconfig

```diff
@@ -104,6 +104,15 @@ config GEN_IRQ_VECTOR_TABLE
 config NUM_IRQS
 	int
 
+config RISCV_PMP
+	bool "Physical Memory Protection support"
+	default y
+	select THREAD_STACK_INFO
+	select MEMORY_PROTECTION
+	select DYNAMIC_OBJECTS
```

Comment:
why select DYNAMIC_OBJECTS? this is a kernel option for extra functionality that shouldn't be required by any arch code

Reply:
My mistake, this was leftover from an earlier test. It's been removed now.